src/objects.h

```diff
@@ -528,6 +528,13 @@ static inline void SET_TYPE_OBJ(Obj obj, Obj type)
 */
 extern void MakeImmutable( Obj obj );
 
+static inline void MakeImmutableNoRecurse(Obj obj)
```

Comment:
I'm a bit worried that this method does quite a lot less than `MakeImmutable`, for example MakeImmutable does things to the various object types.

Is it intended that this should be called on such objects, and not do anything?

Reply:
Just to add: thanks for looking carefully. I should have thought about adding that comment myself